backend/homework/7/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,52 @@
+package org.example;
+
+import org.example.beans.*;
+
+import org.example.config.ProjectConfig;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static org.example.beans.VehicleService.inventory;
+
+public class Main {
+    private static final org.slf4j.Logger logger = LoggerFactory.getLogger(VehicleService.class);
+    public static void main(String[] args) {
+        var context = new AnnotationConfigApplicationContext(ProjectConfig.class);
+
+        Factory factory1 = context.getBean("Factory1", Factory.class);
+        VehicleService vehicleService1 = context.getBean(VehicleService.class);
+
+        vehicleService1.setFactory(factory1);
+        vehicleService1.updateCost(factory1.addTransportationCost());
+
+        logger.info("Vehicles in Factory-1:");
+        vehicleService1.printVehiclesDetails();
+        logger.info("Most expensive vehicle in Factory-1:");
+        vehicleService1.printMostExpensiveVehicle();
+
+
+        Factory factory2 = context.getBean("Factory2", Factory.class);
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.